tests/test_event.py

```diff
@@ -1,22 +1,18 @@
-from fastapi.testclient import TestClient
+class TestEvent:
 
-from app.main import app
+    def test_eventedit(self, client):
+        response = client.get("/event/edit")
+        assert response.status_code == 200
+        assert b"Edit Event" in response.content
 
-client = TestClient(app)
+    def test_eventview_with_id(self, client):
+        response = client.get("/event/view/1")
+        assert response.status_code == 200
+        assert b"View Event" in response.content
 
+    def test_eventview_without_id(self, client):
+        response = client.get("/event/view")
+        assert response.status_code == 404
```

Comment:
Prefer using `HTTP_NOT_FOUND` from `fastapi.status`

Reply:
I used HTTP_404_NOT_FOUND from starlette.status, is it ok?